catalyst-common/src/main/java/org/anvilpowered/catalyst/common/command/MessageCommand.java

```diff
@@ -62,19 +65,35 @@ public int execute(CommandContext<TCommandSource> context, Class<?> consoleClass
                     textService.send(pluginMessages.messageSelf(), context.getSource());
                     return 0;
                 }
-                privateMessageService.sendMessage(
-                    userService.getUserName((TPlayer) context.getSource()),
-                    name,
-                    message
-                );
-                privateMessageService.replyMap().put(
-                    userService.getUUID(targetPlayer.get()),
-                    userService.getUUID((TPlayer) context.getSource())
-                );
-                privateMessageService.replyMap().put(
-                    userService.getUUID((TPlayer) context.getSource()),
-                    userService.getUUID(targetPlayer.get())
-                );
+                Anvil.getServiceManager().provide(CoreMemberManager.class).getPrimaryComponent()
+                        .getOneForUser(
+                                userService.getUUID((TPlayer) context.getSource())
+                        ).thenAcceptAsync(optionalMember -> {
+                    if (!optionalMember.isPresent()) {
+                        return;
+                    }
+                    CoreMember<?> coreMember = optionalMember.get();
+                    if (Anvil.getServiceManager().provide(CoreMemberManager.class).getPrimaryComponent().checkMuted(coreMember)) {
```

Comment:
Honestly, It may be a good idea to store the manager instead of querying anvil twice